sys/include/event/periodic_callback.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2022 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_event
+ * @brief       Provides functionality to trigger periodic event callbacks
+ *
+ * This provides convenience functions to trigger periodic event callbacks
+ * executed by the event thread.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Event Periodic Callback API
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ *
+ */
+
+#ifndef EVENT_PERIODIC_CALLBACK_H
+#define EVENT_PERIODIC_CALLBACK_H
+
+#include "event/callback.h"
+#include "event/periodic.h"
+#include "event/thread.h"
```

Comment:
Same goes for this include btw. I am not sure what of `event/thread.h` is used here.